### PR TITLE
config: enable cherry-pick-approved label to lonng

### PR DESCRIPTION
**Pull request by CadmusJiang**
Signed-off-by: Cadmus <CadmusJiang@gmail.com>

### CODE REVIEW

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -1121,6 +1121,7 @@ ti-community-label-blocker:
           - qa-release-merge
         trusted_users:
           - ti-chi-bot
+          - lonng
```
Comment (hi-rustin) [inline prow/config/external_plugins_config.yaml:1124]: I think you just need to add Longheng for the sig migrate repositories.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: prow/config/external_plugins_config.yaml (hunk 1 above)
Comment (hi-rustin) [reply prow/config/external_plugins_config.yaml:1124]: You can split the repo config.

Comment (CadmusJiang) [reply prow/config/external_plugins_config.yaml:1124]: I see！

Comment (hi-rustin) [reply prow/config/external_plugins_config.yaml:1137]: There are still some problems, each repo can only appear once in repos, so you should copy the complete configuration above before making changes.

**Review by hi-rustin — APPROVED** [review]
/ok-to-test

Thanks!

**Pull request merged**